### PR TITLE
test: testing parallel work in two workspaces with cypress

### PR DESCRIPTION
**Pull request by esasova**
Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

File: cypress/integration/brewery/WorkingInTwoWorkspaces.spec.js [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+        DATASET.BREWERY_ADT,
+        RUN_TEMPLATE.BREWERY_PARAMETERS
+      ).then((value) => {
+        secondWorkspaceParentScenarioId = value.scenarioCreatedId;
+        ScenarioParameters.edit();
+        BreweryParameters.getStockInput().clear().type('400');
+        BreweryParameters.getRestockInput().clear().type('10');
+        BreweryParameters.getWaitersInput().clear().type('15');
+        Workspaces.getHomeButton().should('be.visible').click();
+        cy.go('back');
```
Comment (nborde-CSM) [inline cypress/integration/brewery/WorkingInTwoWorkspaces.spec.js:79]: same as above

File: cypress/integration/brewery/WorkingInTwoWorkspaces.spec.js [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+          firstWorkspaceChildScenarioId = value.scenarioCreatedId;
+          ScenarioParameters.edit();
+          BreweryParameters.switchToCustomersTab();
+          BreweryParameters.importCustomersTableData(CSV_VALID_WITH_EMPTY_FIELDS);
+
+          Workspaces.getHomeButton().should('be.visible').click();
+          Workspaces.selectWorkspace(BREWERY_WORKSPACE_ID);
+
+          // create second child scenario, edit it without launching
+          Scenarios.createScenario(
```
Comment (nborde-CSM) [inline cypress/integration/brewery/WorkingInTwoWorkspaces.spec.js:125]: It is really a good idea to include a scenario creation within another scenario creation promise ?

**Review by nborde-CSM — APPROVED** [review]
Some remarks but LGTM :+1:

File: cypress/integration/brewery/WorkingInTwoWorkspaces.spec.js [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+    ).then((value) => {
+      firstWorkspaceParentScenarioId = value.scenarioCreatedId;
+      ScenarioParameters.getParametersTabs().should('be.visible');
+      ScenarioParameters.edit();
+      BreweryParameters.getStockInput().clear().type('40');
+      BreweryParameters.getRestockInput().clear().type('10');
+      BreweryParameters.getWaitersInput().clear().type('1');
+      ScenarioParameters.updateAndLaunch();
+      Workspaces.getHomeButton().should('be.visible').click();
+      cy.go('back');
```
[the remark this answers is not in the document]
Comment (esasova) [reply cypress/integration/brewery/WorkingInTwoWorkspaces.spec.js:58]: This test serves to check back and forward navigation with browser arrows to be sure that go.back charges previous scenario and so on :slightly_smiling_face:

File: cypress/integration/brewery/WorkingInTwoWorkspaces.spec.js (hunk 2 above)
Comment (esasova) [reply cypress/integration/brewery/WorkingInTwoWorkspaces.spec.js:125]: I'm not sure but is there other way to get scenario id to use it through the test ?

File: cypress/integration/brewery/WorkingInTwoWorkspaces.spec.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (nborde-CSM) [reply cypress/integration/brewery/WorkingInTwoWorkspaces.spec.js:58]: My bad...

File: cypress/integration/brewery/WorkingInTwoWorkspaces.spec.js (hunk 2 above)
Comment (nborde-CSM) [reply cypress/integration/brewery/WorkingInTwoWorkspaces.spec.js:125]: I think we can wrap a value in a promise to use it later but i'm no longer sure...

Comment (csm-thu) [reply cypress/integration/brewery/WorkingInTwoWorkspaces.spec.js:125]: The current implementation seems ok to me, IIRC we use the same nested `createScenario`s in another tests :slightly_smiling_face:  (there is also at least one test where we create scenarios before the `it` block of the test to store all scenarios ids)

Comment (esasova) [reply cypress/integration/brewery/WorkingInTwoWorkspaces.spec.js:125]: yes, in ScenarioLaunch, we create all tests in `before` but in this one, the idea is to perform different consecutive operation including create scenario during the test itself :slightly_smiling_face:

**Pull request merged**